cjsmin/src/chess.ts

```diff
@@ -811,7 +850,7 @@ export class Chess {
     this._board[sq] = {
       type: type as PieceType,
       color: color as Color,
-      unambiguousSymbol: 'pa',
+      unambiguousSymbol,
```

Comment:
not sure i understand this change either. why was it set to 'pa' initially, and why remove this?

Reply:
this is the hardcoding I mentioned. This was done so we could compile while requiring this and left as a TODO For this PR